addons/website_profile/controllers/main.py

```diff
@@ -37,13 +37,17 @@ def _check_avatar_access(self, user_id, **post):
 
     def _check_user_profile_access(self, user_id):
```

Comment:
while we change the method, can we add a docstring ? 
(explain that we either return the user or the reason why we can't access it)